filterlist/imageboard/GelboLube.txt

```diff
@@ -90,6 +90,12 @@ gelbooru.com###navbar ul.nav li:has(a[href*="moneygrubbingwhore.com"])
 gelbooru.com###navbar ul.nav li:has(a[href*="gydadc.com"])
 gelbooru.com###navbar ul.nav li:has(a[href*="thedoujin.com"])
 
+! 10. Remove blur overlay for images requiring an account
+gelbooru.com##body img:style(filter:none !important;)
```

Comment:
Must be move to [this section](https://github.com/kowith337/PersonalFilterListCollection/blob/3399329db8a1f2f401646901b50125327c5edd00/filterlist/imageboard/GelboLube.txt#L307) instead because this rule will replace CSS property

However, I will merge and re-commit again, hope you didn't mind.

Reply:
Ah, wasn't sure about the exact location. Happy for you to rearrange the code as you see fit. Glad I could help!